src/client/js/components/circle-chart.js

```diff
@@ -0,0 +1,244 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const CHART_RADIUS = 50
+const CHART_DIAMETER = CHART_RADIUS * 2
+const CHART_CIRCUMFERENCE = Math.PI * CHART_DIAMETER
+const CHART_UPDATE_DURATION = 250
+
+const styles = `
+<style>
+  .circle-chart {
+    align-items: center;
+    display: flex;
+    gap: var(--padding-md);
+    margin: 0;
+    transition: opacity ${CHART_UPDATE_DURATION * 0.5}ms ease;
+  }
+
+  .circle-chart.updating {
+    opacity: 0
+  }
+
+  .circle-chart-title {
+    display: block;
+    font-family: Inter, Inter-fallback, sans-serif;
```

Comment:
I had issues with the font sizes when I attempted to add the component to the breaches page: the labels were larger than the heading.  Styles in web components are tricky – we can finesse this when we add it to the breaches page.

Reply:
Oh. Yes, let’s look into this when adding the charts to the view they are intended for.